### ISSUE TITLE
Problem Re-Installing Sickchill w/ Pip

### ISSUE THREAD

**Issue by vjbonan** [SickChill/sickchill#8310]
OS:  Ubuntu 18.04.6 LTS 
What you did:     Rebooted
What happened: Sickchill stopped working
What you expected:  Sickchill to start 

Hello,  I had a Sickchill installed via an old install not in pip   I am trying to reinstall Sickchill with PIP, following the following thread, however,  it fails.
https://github.com/SickChill/SickChill/wiki/Pip#ubuntudebian

After installing Python 3.8, (Also tried 3.7)

Namely these steps:  
DIR=/opt/sickchill
[ -d "${DIR}" ] && sudo mv ${DIR} ${DIR}_old
python3 -m venv $DIR
${DIR}/bin/pip install -U sickchill <--------------
${DIR}/bin/SickChill --data_dir.....

When issuing: ${DIR}/bin/pip install -U sickchill
I receive:
```
  Using cached https://files.pythonhosted.org/packages/3f/06/89b92a27fe2e5be1519b268ec5b92eabcaa7421a158c3ad874b439d0f0ef/pymediainfo-5.1.0.tar.gz
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-build-umtrm5zd/pymediainfo/setup.py", line 70, in <module>
        "License :: OSI Approved :: MIT License",
      File "/opt/sickchill/lib/python3.6/site-packages/setuptools/__init__.py", line 129, in setup
        return distutils.core.setup(**attrs)
      File "/usr/lib/python3.6/distutils/core.py", line 108, in setup
        _setup_distribution = dist = klass(attrs)
      File "/opt/sickchill/lib/python3.6/site-packages/setuptools/dist.py", line 372, in __init__
        _Distribution.__init__(self, attrs)
      File "/usr/lib/python3.6/distutils/dist.py", line 281, in __init__
        self.finalize_options()
      File "/opt/sickchill/lib/python3.6/site-packages/setuptools/dist.py", line 528, in finalize_options
        ep.load()(self, ep.name, value)
      File "/opt/sickchill/lib/python3.6/site-packages/pkg_resources/__init__.py", line 2324, in load
        return self.resolve()
      File "/opt/sickchill/lib/python3.6/site-packages/pkg_resources/__init__.py", line 2330, in resolve
        module = __import__(self.module_name, fromlist=['__name__'], level=0)
      File "/tmp/pip-build-umtrm5zd/pymediainfo/.eggs/setuptools_scm-7.0.5-py3.6.egg/setuptools_scm/__init__.py", line 5
        from __future__ import annotations
        ^
    SyntaxError: future feature annotations is not defined
    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-build-umtrm5zd/pymediainfo/
```

Any help would be appreciated.  Thanks in advance!

**Comment by miigotu** [comment]
Your venv has a python2 version installed. Delete the venv and start over.

**Comment by vjbonan** [comment]
Hmm I updated the command in the FAQ to:
**_sudo apt install python3.9 python3.9-venv python3-pip_**

Is there a python3.9-pip?, when attempting that like the other 2, it  wasn't found.

Then I deleted the directory created from  **_python3 -m venv $DIR_**

and then ran the install again  **_${DIR}/bin/pip install -U sickchill ._** and received the same error
Similar error, showing pythong 3.6?
.
.
.
  File "/opt/sickchill/lib/python3.6/site-packages/setuptools/__init__.py", line 129, in set


it's like when I installed python 3.9, its not a default??.. I'm far from an expert here so sorry for the layman's terms.   

I did have to change the alias, 
alias python3=python3.9
I confirmed python3 --version was showing python 3.9

ie
username@system:/opt$ python3 --version
Python 3.9.15
username@system:/opt$

What am I missing here?

Thanks in advance!

**Comment by TFWulp** [comment]
Try using pip3, `${DIR}/bin/pip3 install -U sickchill`. If that doesn't work, on some unix OS's you have pip3.9 so it becomes `${DIR}/bin/pip3.9 install -U sickchill`

**Comment by vjbonan** [comment]
All rightie, it was the venv environment, changing the command to 

python -m venv $DIR  to **python3.7 -m venv $DIR**

allowed it to continue,,,, 
But there's a NEW problem ... nothing's easy.    Install still fails,  It says check permissions, but this file simply does  NOT exist....

```
Collecting pycparser
  Using cached pycparser-2.21-py2.py3-none-any.whl (118 kB)
Installing collected packages: win-inet-pton, timeago, text-unidecode, Send2Trash, rebulk, rarfile, pytz, PyNMA, profilehooks, msgpack, jsonrpclib-pelix, ipaddress, ifaddr, gntp, enzyme, deluge-client, bencode.py, appdirs, xmltodict, wrapt, urllib3, Unidecode, tornado, soupsieve, six, python-slugify, pyparsing, pyjwt, pycparser, pbr, oauthlib, MarkupSafe, markdown2, lxml, imagesize, idna, greenlet, future, decorator, click, charset-normalizer, chardet, certifi, babelfish, validators, stevedore, SQLAlchemy, requests, python-dateutil, pysrt, packaging, Mako, deprecated, configobj, cffi, beekeeper-alt, beautifulsoup4, tvdbsimple, tus.py, tmdbsimple, requests-oauthlib, qbittorrent-api, python3-fanart, pynacl, new-rtorrent-python, kodipydent-alt, guessit, dogpile.cache, cryptography, cinemagoer, CacheControl, subliminal, python-twitter, pyOpenSSL, PyGithub, putio.py, IMDbPY, sickchill
ERROR: Could not install packages due to an OSError: [Errno 13] Permission denied: '/opt/sickchill/lib/python3.11/site-packages/win_inet_pton.py'
Check the permissions.


[notice] A new release of pip available: 22.3 -> 22.3.1
[notice] To update, run: python3.11 -m pip install --upgrade pip
```
What do you folks think will get past this one?  

Thanks in advance...

**Comment by vjbonan** [comment]
I was able to muddle through the other missing packages.   Finally got it running. 

Thx for the help
